IFC4x3/ModelViews/General Usage/DocModelView.xml

```diff
@@ -10514,6 +10514,13 @@ Figure 1 illustrates project relationships with spatial structures, elements, an
 						<DocTemplateItem UniqueId="e1170aa4-751f-4de7-b936-8a1a853c30d7" />
 					</Items>
 				</DocTemplateUsage>
+				<DocTemplateUsage Name="Project Global Positioning Mapped" UniqueId="225782d2-563c-445b-a7c5-396743a56a6b">
+					<Documentation>The representation context of the project refers to a global positioning, i.e. the local engineering coordinate system of the project has a mapping, including a rigid transformation, to a defined projected coordinate system (a rectangular map coordinate system, as used in GIS systems)</Documentation>
```

Comment:
<Documentation>The representation context of the project refers to global positioning, i.e. the coordinate system of the project resides within the defined projected coordinate reference system (a map coordinate system) and is subject to a rigid transformation, e.g. to achieve smaller coordinate values. The representation context inherits all properties of the projected coordinate reference system, in particular the distortions resulting in deviations between model dimensions/distances and real-world dimensions/distances.</Documentation>

Reply:
Polished the wording again...